extensions/elytron-security/runtime/pom.xml

```diff
@@ -12,7 +12,7 @@
 
     <artifactId>quarkus-elytron-security</artifactId>
     <name>Quarkus - Elytron Security - Runtime</name>
-    <description>Secure your services</description>
+    <description>Secure your services via Elytron</description>
```

Comment:
What do we mean by services here?

Reply:
No idea, I asked wehther that extension should even be visible but only DAvid replied "I don't know"